resources/assets/js/components/user-dashboard.vue

```diff
@@ -1,3 +1,4 @@
+<!--suppress UnnecessaryLabelJS -->
```

Comment:
oops, that must have been a weird phpStorm thing that slipped in there. Nice catch.